Source/Diagnostics/ReducedDiags/FieldProbe.H

```diff
@@ -100,6 +103,11 @@ private:
     //! Judges whether to gather raw fields or interpolated data
     bool raw_fields = false;
 
+    //! Judges whether to follow a moving window
+    bool do_moving_window_FP = false;
+
+    //! Calculates particle movement in moving window sims
+    amrex::Real temp_dist_mult;
```

Comment:
I think this can be a local variable inside `FieldProbe::ComputeDiags` and does not need to be stored here on the class level as a member variable.
```suggestion
```